configtx/consortiums_test.go

```diff
@@ -1486,6 +1486,60 @@ func TestRemoveConsortiumOrg(t *testing.T) {
 	gt.Expect(c.Consortium("Consortium1").Organization("Org1")).To(BeNil())
 }
 
+func TestSetConsortiumOrgModPolicy(t *testing.T) {
+	t.Parallel()
+
+	gt := NewGomegaWithT(t)
+
+	consortiums, _ := baseConsortiums(t)
+
+	consortiumsGroup, err := newConsortiumsGroup(consortiums)
+	gt.Expect(err).NotTo(HaveOccurred())
+
+	config := &cb.Config{
+		ChannelGroup: &cb.ConfigGroup{
+			Groups: map[string]*cb.ConfigGroup{
+				ConsortiumsGroupKey: consortiumsGroup,
+			},
+		},
+	}
+
+	c := New(config)
+
+	consortium1Org1 := c.Consortium("Consortium1").Organization("Org1")
+	err = consortium1Org1.SetModPolicy("TestModPolicy")
+	gt.Expect(err).NotTo(HaveOccurred())
+
+	updatedModPolicy := consortium1Org1.orgGroup.GetModPolicy()
+	expectedModPolicy := "TestModPolicy"
```

Comment:
nit: literal in the assertion; no need for temporary.

This applies here and for the remaining assertions with string assignment to a temporary.